src/content/content.ts

```diff
@@ -20,20 +20,19 @@
  * SOFTWARE.
  */
 
-import Action from "@web-eid/web-eid-library/models/Action";
-import ContextInsecureError from "@web-eid/web-eid-library/errors/ContextInsecureError";
+import Action from "@web-eid.js/models/Action";
+import ContextInsecureError from "@web-eid.js/errors/ContextInsecureError";
 
-import config from "../config";
-import HttpResponse from "../models/HttpResponse";
 import { TokenSigningErrorResponse } from "../models/TokenSigning/TokenSigningResponse";
-import { headersToObject } from "../shared/utils";
-import tokenSigningResponse from "../shared/tokenSigningResponse";
+import config from "../config";
 import injectPageScript from "./TokenSigning/injectPageScript";
+import tokenSigningResponse from "../shared/tokenSigningResponse";
 
 function isWebeidEvent(event: MessageEvent): boolean {
   return (
     event.source === window &&
-    event.data?.action?.startsWith?.("web-eid:")
+    event.data?.action?.startsWith?.("web-eid:") &&
+    event.data.action !== Action.WARNING
```

Comment:
Moved the warning message check to another part of the code, because the warning message actually is a Web-eID event and so it shouldn't be checked for in the "isWebeidEvent" function.
Also commented why it's needed.